README.md

```diff
@@ -283,7 +283,8 @@ And finally, you are ready to go! Or at least you should be...
 
 ### Junior Contributors
 
-
+**Alexandra Caus** | [alexcaus03][alexcaus03]
+[alexcaus03]: <https://github.com/alexcaus03>
```

Comment:
Almost got it!!

Just a few notes before finishing! After you put your **nickname**, you must place another | and put your phrase. Also carefull when placing the link cause it has to be at the bottom part with the rest of links, which are sorted alfabetically.

After this changes are applied that should be finished!!